docs/tutorials/training_transformers.md

```diff
@@ -0,0 +1,93 @@
+# Training Transformers
+
+This tutorial shows how to train a transformer policy network.
+
+<div class="admonition note">
+<p class="admonition-title">Note: Understand how to launch training runs and view results first!</p>
+
+Before trying to train a transformer, it might be useful to read the following tutorials:
+- [how to launch training runs](./configs.html)
+- [how to view training results](./viewing_results.html)
+- [how to launch multiple training runs efficiently](./hyperparam_scan.html) 
+
+</div>
+
+A template with tuned parameters for transformer based policy networks is defined in `robomimic/config/default_templates/bc_transformer.json`.
+
+#### 1. Using default configurations
+
+The easiest way to train a transformer policy network is to pass the default template json to the main training script `train.py` via the `--config` argument. The dataset can be specified by setting the `data` attribute of the `train` section of the config json, or specified via the `--dataset` argument.
+
+```sh
+$ python train.py --config ../config/default_templates/bc_transformer.json --dataset /path/to/dataset.hdf5
+```
+
+#### 2. Defining custom parameters
+
+If you want to modify the default transformer parameters, do not directly modify the default config (`config/bc_config.py`) or template (`config/default_templates/bc_transformer.json`).  Instead, you should modify the default template in python code and then call the train function, as shown in the code snippet below.
+
+This code highlights useful parameters to tune for transformers and modifications that should be made when training a policy on observation images (default is low-dimensional states).  To see all transformer policy settings, refer to `config/bc_config.py`.
+
+```python
+import json
+import robomimic
+import robomimic.utils.torch_utils as TorchUtils
+from robomimic.config import config_factory
+from robomimic.scripts.train import train
+
+# load json with default transformer parameters
+ext_cfg = json.load(open('robomimic/config/default_templates/bc_transformer.json', 'r'))
+#generate base behavioral cloning config
+config = config_factory(ext_cfg["algo_name"])
+
+with config.values_unlocked():
+    # update config with default transformer parameters
+    config.update(ext_cfg)
+
+    # set config attributes for training
+    config.experiment.name = "bc_transformer_example"
+    config.train.data = "/path/to/dataset.hdf5"
+    config.train.output_dir = "/path/to/output_dir/"
+
+    # set horizon to be greater than the length of your longest demonstration
```

Comment:
this isn't a hard and fast rule - just say set it appropriately based on the length of demonstrations in your dataset (which can be obtained with `scripts/get_dataset_info.py`)